scripts/profiler/stats_nordic.py

```diff
@@ -53,16 +55,45 @@ def _get_timestamps(self, event_name, event_state):
             return None
 
         if event_state == EventState.SUBMIT:
-            timestamps = list(map(lambda x: x.submit.timestamp, trackings))
+            timestamps = np.fromiter(map(lambda x: x.submit.timestamp, trackings),
+                                     dtype=np.float)
         if event_state == EventState.PROC_START:
-            timestamps = list(map(lambda x: x.proc_start_time, trackings))
+            timestamps = np.fromiter(map(lambda x: x.proc_start_time, trackings),
+                                     dtype=np.float)
         if event_state == EventState.PROC_END:
-            timestamps = list(map(lambda x: x.proc_end_time, trackings))
+            timestamps = np.fromiter(map(lambda x: x.proc_end_time, trackings),
+                                     dtype=np.float)
 
+        timestamps = np.fromiter(filter(lambda x: x > start_meas
+                                        and x < end_meas,
+                                        timestamps), dtype=np.float)
         return timestamps
 
+    def calculate_times_between(self, start_times, end_times):
+        if end_times[0] <= start_times[0]:
+            end_times = end_times[1:]
+        if len(start_times) > len(end_times):
+            start_times = start_times[:-1]
+
+        return (end_times - start_times) * 1000
+
+    def prepare_stats_txt(self, times_between):
+        stats_text = "Max time: "
+        stats_text += "{0:.3f}".format(max(times_between)) + "ms\n"
+        stats_text += "Min time: "
+        stats_text += "{0:.3f}".format(min(times_between)) + "ms\n"
+        stats_text += "Mean time: "
+        stats_text += "{0:.3f}".format(np.mean(times_between)) + "ms\n"
+        stats_text += "Std of time: "
```

Comment:
Text should be `Std dev` for standard deviation

Reply:
done